cypress/support/commands.ts

```diff
@@ -51,15 +58,11 @@ Cypress.Commands.add(
     cy.visit(Cypress.env("baseUrl"));
     cy.get("[data-cy=file-dropdown]").click();
     cy.get('[data-cy="file-input"]').attachFile(scheduleName);
-    cy.get(`[data-cy=nurseShiftsTable]`, { timeout: 10000 });
+    cy.get(`[data-cy=nurseShiftsTable]`).should("exist");
```

Comment:
Podłączam dwa wpisy ciekawe ze stack overflow, to tym o czym rozmawialiśmy na spotkaniu:
https://stackoverflow.com/questions/50042670/cypress-test-is-contains-equivalent-to-shouldcontain
https://stackoverflow.com/questions/57054406/is-the-shouldexist-assertion-redundant-on-cypress

Reply:
Właśnie zobaczyłem na dashboardzie, że powtarzają się nie testy, a tylko ostatnia komenda.
Nie wiem czy to widać na obrazku
![image](https://user-images.githubusercontent.com/21079319/107863771-37a98a00-6e57-11eb-8c8c-faa29e6f277d.png)

Dzięki że podesłałeś. 
Z tego co rozumiem, to główna różnica jest taka, że po `contains` my możemy dołączyć jeszcze kontynuować nasz `chain`, a po `should` nie.

Ale w tym przypadku to akurat nie potrzebujemy dalej to chainować, więc myślę, że zostawię tę asercję z `should`.